domains/Auth/SignIn/index.js

```diff
@@ -57,6 +59,14 @@ const SignIn = ({ navigation }) => {
     });
   }, [load]);
 
+  const handleFailedAttempt = (err) => {
```

Comment:
decided to use an Alert because rendering any modal view didn't work that nested in that callback hell